Shared/Samples/Select features in scene layer/README.metadata.json

```
@@ -0,0 +1,24 @@
+{
+    "category": "Scenes",
+    "description": "Identify features in a scene to select.",
+    "ignore": false,
+    "images": ["select-features-in-scene.png"],
```

```suggestion
    "images": [
        "select-features-in-scene.png"
    ],
```
We prefer put the filename on its own line so future changes can be limited to each line.